sdk/core/Azure.Core/tests/public/PatchModels/ParentPatchModel.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Json;
+
+namespace Azure.Core.Tests.PatchModels
+{
+    /// <summary>
+    /// This model illustrates a patch model with properties that are nested models.
+    /// </summary>
+    public partial class ParentPatchModel
+    {
+#pragma warning disable AZC0020 // Avoid using banned types in libraries
+        private readonly MutableJsonElement _element;
+
+        /// <summary>
+        /// Public constructor.
+        /// </summary>
+        public ParentPatchModel()
+        {
+            _element = MutableJsonDocument.Parse(MutableJsonDocument.EmptyJson).RootElement;
+        }
+
+        /// <summary>
+        /// Serialization constructor.
+        /// </summary>
+        /// <param name="element"></param>
+        internal ParentPatchModel(MutableJsonElement element)
+        {
+            _element = element;
+        }
+
+        /// <summary>
+        /// Optional string property corresponding to JSON """{"id": "abc"}""".
+        /// </summary>
+        public string Id
```

Comment:
I would add some readonly properties somewhere to validate that portion of the expected serialization stuff

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/38159/commits/1e08e78aa58b488d359a8e261b0f67ffc3508c45